pkg/shim/runsc/BUILD

```diff
@@ -31,6 +31,7 @@ go_library(
         "@com_github_containerd_cgroups//:go_default_library",
         "@com_github_containerd_cgroups//stats/v1:go_default_library",
         "@com_github_containerd_cgroups//v2:go_default_library",
+        "@com_github_containerd_cgroups//v2/stats:go_default_library",
```

Comment:
our internal pipeline is missing the rule to translate the package, it blocks the submission.

i will fix that in our pipeline